script.js

```diff
@@ -2,11 +2,71 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  console.log(allEpisodes);
 }
-
+ const rootElem = document.getElementById("root");
+// show all episod lists in main page
 function makePageForEpisodes(episodeList) {
```

Comment:
this is a nice clear function - all your episodes show on initial load, well done :)